images/ose-machine-config-operator.yml

```diff
@@ -9,7 +9,7 @@ content:
     ci_alignment:
       streams_prs:
         ci_build_root:
-          stream: rhel-9-golang-ci-build-root
+          stream: rhel-8-golang-ci-build-root
```

Comment:
Think this should be rhel9. 
It is info used for constructing the alignment PRs, and is not used for anything else. It sdhould therefore express our wished for the future.

Reply:
```suggestion
          stream: rhel-9-golang-ci-build-root
```